administration/src/bp-modules/statistics/constants/index.ts

```diff
@@ -0,0 +1,6 @@
+import { subYears } from 'date-fns'
+
+import PlainDate from '../../../util/PlainDate'
+
+export const defaultStartDate = PlainDate.fromLocalDate(subYears(new Date(), 1)).toString()
```

Comment:
This does not work for Schaltjahre  :D But yeah... It's not super important :D

Reply:
ok then 
```js
// subtract a year from now and add a day to have exactly one year, since end date is included in time period. this also considers leap years
export const defaultStartDate = PlainDate.fromLocalDate(addDays(subYears(new Date(), 1),1)).toString()
```